tsconfig.json

```diff
@@ -14,6 +14,7 @@
     "strictNullChecks": true,
     "noImplicitReturns": true,
     "strictFunctionTypes": true,
+    "strictPropertyInitialization": true,
```

Comment:
My assumption is that this controls all compilations as I see all the other ones extending this one. 

However, I also had some merge conflicts while exporting the change from g3, so I know that there are missing ! in src/material/menu/menu.ts. However, CI builds? What gives?